### ISSUE TITLE
backup/restore workspace to original monitor

### ISSUE THREAD

**Issue by aarobc** [JonnyHaystack/i3-resurrect#101]
**Is your feature request related to a problem? Please describe.**
Currently the only way to assign a workspace to a monitor is to hardcode it within the i3 config file. This is inconvenient as it limits the portability of the i3 config between machines with different display configurations.

**Describe the solution you'd like**
Upon saving a workspace layout, the current monitor is recorded under an additional property.

Upon restoring a workspace layout, it's created on the monitor configured in the layout file.

**Describe alternatives you've considered**
Hardcoding the workspace within the i3 config. This is much too limiting.

**Comment by JonnyHaystack** [comment]
Why not just assign your workspaces to monitors in the i3 config?

**Comment by aarobc** [comment]
You're talking about assigning the workspace permanently. My request was to do a one time "move workspace to monitor" step on only when the restore is run. I feel like this would be more consistent with what a user would expect. 

You could include the output within the generated config, and upon restore, preform something like
```
i3.workspace(str(workspace))                                  
i3.command('move', 'workspace to output ' + output['name']) 
```

This would accommodate a much broader range of i3 users, and not require dynamically generating the i3 config (which I'm averse to).

**Comment by JonnyHaystack** [comment]
Okay, I get you. i3-resurrect is generally designed around not having to manually edit the layout files, but this seems like a good feature so I'm happy to give that option in the case that you do want to manually add it.

**Comment by aarobc** [comment]
I apologize for the confusion, but my request hopes to avoid any manual editing at all:

- the layout saving also includes the workspace display as a property in the saved layout file
- the restore process consumes the property and places the workspace on the stored property value

**Comment by JonnyHaystack** [comment]
Right, I can make a command line flag for that then, seeing as people might not want that behaviour by default

**Comment by moodboom** [comment]
This would be great.  Without this feature, i cannot simply "restore all" desktops via a simple script, as suggested [here](https://github.com/JonnyHaystack/i3-resurrect/issues/84#issuecomment-711016873).

**Comment by kucaiyi** [comment]
This could be close by the commit 61f27f4dc877f089bffbe87b4440acd663cd0952
